### DISCUSSION TITLE
Corr.grd having large blanks in Radar Coordinates

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#857] [Q&A]
Having an interesting effect when looking at the corr.grd files of some of the interferograms.  Having these large blank spaces.  I have not seen anything like this happen before in my limited experience, and the correlation stack does not like this, obviously.  Seems to look okay when I project it to lat lon, but not sure what is cooking here to cause this, or if its a problem.  

I was getting failures during unwrapping, so started back tracking looking for oddities and stumbled into this when the corr grids seemed to be heavily masked, and giving me this error when setting up for snaphu:

interpolating masked values using a nearest neighbor algorithm
Namespace(infile='phase_patch_mask.grd', outfile='phase_patch_interp.grd', plots=False, lonlat=False, ncconvert=True)
ERROR 4: `phase_patch_interp.grd' not recognized as a supported file format.

![image](https://github.com/gmtsar/gmtsar/assets/41383336/5a1630c6-c6f7-4829-9f95-91c9014eaff6)

**Comment by Xiaohua-Eric-Xu** [comment]
This could be that there are a few bursts offset between your reference and repeat images. e.g., your reference covers burst 1-10, but your repeat covers 4-13 (burst numbers).

**Reply by krkrueger** [reply]
Ah, interesting, have never seen something like this before.  This of course causes issue if I try to do a stacked correlation mask, I end up with blanks at the bottom and at the top.  Any good ways to get around this?  Do you need to geocode first... before masking and then unwrapping? Maybe some kind of way of automatically leaving out these particular grds in the stack corr?

But ultimately, this doesn't necessarily mean there is a mistake somewhere correct? Just need to figure out how to deal with it. 

As always, thanks for the input.

**Reply by Xiaohua-Eric-Xu** [reply]
It is always recommended to run the organizing scripts. i.e., organize_files_tops[_linux].csh to reassemble the data into new "frames". In that case, if any dates lack data coverage, it'll skip that date.
